codebuild/release/release-prod.yml

```diff
@@ -11,8 +11,8 @@ env:
   secrets-manager:
     GPG_KEY: Maven-GPG-Keys-Release-Credentials:Keyname
     GPG_PASS: Maven-GPG-Keys-Release-Credentials:Passphrase
-    SONA_USERNAME: Sonatype-Team-Account:Username
-    SONA_PASSWORD: Sonatype-Team-Account:Password
+    SONA_USERNAME: Sonatype-User-Token:username
+    SONA_PASSWORD: Sonatype-User-Token:password
```

Comment:
? I'm confused
Shouldn't this be replaced with just the token? Are you moving the creds from one secret to another? Or what exactly is this change?

I imagine we would want to add a `SONA_TOKEN: Sonatype-User-Token:token`

Reply:
No, read the instructions for Sonatype User Tokens carefully.
See: https://central.sonatype.org/publish/generate-token/

You can see they have two fields.

Reply:
ahhh, this pr description does not have this link, so it makes it seem that we are just renaming stuff